requirements/requirements-dev.txt

```diff
@@ -4,182 +4,182 @@
 #
 #    './deps compile' (for details see README)
 #
-alabaster==0.7.12
-aniso8601==7.0.0
+alabaster==0.7.12         # via -r /home/karl/raiden/requirements/requirements-docs.txt, sphinx
```

Comment:
Can we somehow avoid having anyone's username in there? Otherwise the diff will be this big for every change from different people.

Reply:
Uh, that's wrong of course! Why did it put the full path in there? I'll see if I can change that.

Reply:
Looks like [others have the same problem](https://github.com/jazzband/pip-tools/issues/1084). Why does it work for dependabot, see https://github.com/raiden-network/raiden/pull/5986?

Reply:
I assume the reason is that our wrapper script (that handles the dependency resolution between the files) passes absolute paths. 

I'll see if that is easily fixable.